index.js

```diff
@@ -39,14 +39,12 @@ class DHT extends EventEmitter {
 
     this.concurrency = opts.concurrency || 10
     this.bootstrapped = false
-    this.ephemeral = opts.id ? !!opts.ephemeral : true
+    this.ephemeral = true
     this.firewalled = this.io.firewalled
     this.adaptive = typeof opts.ephemeral !== 'boolean' && opts.adaptive !== false
     this.destroyed = false
 
     this._nat = new NatSampler()
-    this._port = opts.port || 0
-    this._host = opts.host || '0.0.0.0'
```

Comment:
This doesn't seem to be used anywhere, because it's repeated on `io` which is where it's used